config/locales/es.yml

```diff
@@ -21,3 +21,11 @@ es:
       welcome_to_daria: "¡Bienvenido a DARIA (edición de %{fund})!"
       what_line: "¿En cuál línea estás trabajando?"
       start: "Empezar"
+  devise:
+    sessions:
+      new:
+        google_signin: "Inicia sesión con Google (recomendado)"
+        password_signin: "O, inicie sesión con una contraseña:"
```

Comment:
In some Spanish cultures I think it's preferred to use "clave" vs "contraseña" to refer to a typed password (at least from when I was living in Argentina). Not 100% sure here, but I figured I'd flag it for the expert @montanezp

Reply:
I've seen both used though I would suggest "contraseña" here as, at least anecdotally, it's been used for "password" more.